src/AltinnCore/Common/Services/Implementation/GiteaAPIWrapper.cs

```diff
@@ -265,6 +272,53 @@ public async Task<string> CreateAppToken(string name)
             }
         }
 
+
+        /// <summary>
+        /// List app tokens for a user. Warning there is talks about removing this.
+        /// </summary>
+        /// <param name="userName">The user name</param>
+        /// <param name="password">The password</param>
+        /// <returns>The sha1 value</returns>
+        public async Task<List<AltinnCore.RepositoryClient.Model.AccessToken>> ListAccessTokens(string userName, string password)
+        {
+            List<AltinnCore.RepositoryClient.Model.AccessToken> accessTokens = null;
+            DataContractJsonSerializer serializer = new DataContractJsonSerializer(typeof(List<AltinnCore.RepositoryClient.Model.AccessToken>));
+
+            string giteaSession = AuthenticationHelper.GetGiteaSession(_httpContextAccessor.HttpContext, _settings.GiteaCookieName);
+            User user = GetCurrentUser(giteaSession).Result;
+
+            Uri giteaUrl = new Uri(_settings.ApiEndPoint + "/users/" + user.Login + "/tokens");
+            Cookie cookie = new Cookie(_settings.GiteaCookieName, giteaSession, "/", _settings.ApiEndPointHost);
+            if (Environment.GetEnvironmentVariable("GiteaApiEndpoint") != null && Environment.GetEnvironmentVariable("GiteaEndpoint") != null)
+            {
+                giteaUrl = new Uri(Environment.GetEnvironmentVariable("GiteaApiEndpoint") + "/users/" + user.Login + "/tokens");
+                cookie = new Cookie(_settings.GiteaCookieName, giteaSession, "/", Environment.GetEnvironmentVariable("GiteaEndpoint"));
+            }
+
+            using (HttpClient client = new HttpClient())
+            {
+                byte[] byteArray = Encoding.ASCII.GetBytes(userName + ":" + password);
+                client.DefaultRequestHeaders.Authorization = new System.Net.Http.Headers.AuthenticationHeaderValue("Basic", Convert.ToBase64String(byteArray));
+
+                HttpResponseMessage response = await client.GetAsync(giteaUrl);
+                if (response.StatusCode == System.Net.HttpStatusCode.OK)
+                {
+                    Stream stream = await response.Content.ReadAsStreamAsync();
+                    accessTokens = serializer.ReadObject(stream) as List<AltinnCore.RepositoryClient.Model.AccessToken>;
+                    return accessTokens;
+                }
+                else if (response.StatusCode == System.Net.HttpStatusCode.Forbidden ||
+                response.StatusCode == System.Net.HttpStatusCode.Unauthorized)
```

Comment:
cant we rewrite it like 
else
{
   return null;
}
because we are returning the same outside the else if